aws/ecs/statuser.go

```diff
@@ -130,39 +139,65 @@ func (s Statuser) StatusOverview(ctx context.Context) (any, error) {
 
 func (s Statuser) Status(ctx context.Context) (any, error) {
 	st := Status{Tasks: make([]StatusTask, 0)}
-	if s.Infra.ServiceName == "" {
-		// TODO: Add support for Nullstone tasks (apps that aren't long-running)
-		return st, nil
-	}
-
-	svcHealth, err := GetServiceHealth(ctx, s.Infra)
+	tasks, err := s.getTasks(ctx)
 	if err != nil {
 		return st, err
 	}
 
-	tasks, err := GetServiceTasks(ctx, s.Infra)
+	svcHealth, err := GetServiceHealth(ctx, s.Infra)
 	if err != nil {
 		return st, err
-	} else if len(tasks) > 0 {
-		return st, nil
 	}
 
+	taskDefs := map[string]*ecstypes.TaskDefinition{}
 	for _, task := range tasks {
+		var taskDef *ecstypes.TaskDefinition
```

Comment:
While it's probable that `taskDef` never ends up nil, it would be better practice to define this as:
```suggestion
		var taskDef ecstypes.TaskDefinition
```
This would ensure that we cannot possibly reach a nil panic later in code.

Reply:
I'm not sure how this would work. taskDef can be nil. If it is nil, I don't want to return, I can keep processing. The only impact it would have is that I can't collect port mappings. The case where the taskDef is nil is handled in `findContainerDefinition`.